tool/tsh/tsh.go

```diff
@@ -1493,7 +1497,7 @@ func onLogin(cf *CLIConf) error {
 			if err != nil {
 				return trace.Wrap(err)
 			}
-			if err := updateKubeConfigOnLogin(cf, tc, ""); err != nil {
+			if err := updateKubeConfigOnLogin(cf, tc, "", updateKubeConfigOption); err != nil {
```

Comment:
nit: I notice we never set `path` to anything besides `""`, it can be removed from `updateKubeConfigOnLogin` and `warnOnDeprecatedKubeConfigServerName`

Reply:
done